src/core/object.cc

```diff
@@ -492,11 +492,7 @@ CL_DEFUN bool core__sl_boundp(T_sp obj) {
 void General_O::describe(T_sp stream) { clasp_write_string(this->__repr__(), stream); }
 
 void General_O::__write__(T_sp strm) const {
-  if (_sym_STARliteral_print_objectSTAR->symbolValue().notnilp()) {
```

Comment:
as discussed on IRC - if you don't mind nuking this "literal print object" thing too, go ahead and do that